draft-ietf-add-ddr.md

```diff
@@ -471,11 +473,10 @@ To limit the impact of discovery queries being dropped either maliciously or
 unintentionally, clients can re-send their SVCB queries periodically.
 
 {{Section 8.2 of !I-D.ietf-add-svcb-dns}} describes a second downgrade attack
-where an attacker can block connections to the encrypted DNS server,
-and recommends that clients prevent it by switching to SVCB-reliant behavior once
-SVCB resolution does succeed. For DDR, this means that once a client discovers
-a compatible Designated Resolver, it SHOULD NOT use unencrypted DNS until the
-SVCB record expires, unless verification of the resolver fails.
+where an attacker can block connections to the encrypted DNS server. For DDR,
+clients need to validate a Designated Resolver using a connection to the
+server before trusting it, so attackers that can block these connections can
+prevent clients from switching to use encrypted DNS.
```

Comment:
@mstojens check this rework of the paragraph. I now still mention the attack, and just mention that it means that this is a way you can be downgraded. (Talked to Ben about this)

Reply:
LGTM